src/components/Dex.vue

```diff
@@ -86,45 +86,40 @@ export default {
       // this.$services.neo.resetSystemAssetBalanceCache();
     });
 
-    store.commit('setSocketOrderCreationFailed', (message) => {
+    this.$store.commit('setSocketOrderCreationFailed', (message) => {
       services.alerts.error(`Failed to Create ${(message.side === 'bid' ? 'Buy' : 'Sell')} Order. ${message.data.errorMessage}`);
       // services.neo.resetSystemAssetBalanceCache();
     });
 
-    store.commit('setSocketOrderMatchFailed', (message) => {
+    this.$store.commit('setSocketOrderMatchFailed', (message) => {
       services.alerts.error(`Failed to Match ${(message.side === 'bid' ? 'Buy' : 'Sell')} x${message.data.quantity}. ${message.data.errorMessage}`);
       // services.neo.resetSystemAssetBalanceCache();
     });
 
-    services.neo.promptGASFractureIfNecessary();
+    this.$services.neo.promptGASFractureIfNecessary();
```

Comment:
GAS fracture form shows up now on mobile?

Reply:
This is just copied from desktop./

Reply:
Experience may be poor with fees set for some users without the gas fracture.